### PR TITLE
fix: references updated to stac.linz.govt.nz and extension fixes - IGNORE

### PR DESCRIPTION
**Pull request by amfage**
**Changes:**

- Update all references to linz.github.io/stac to be stac.linz.govt.nz.
- Do not require eo:bands asset field in Historical Imagery extension.
- Update example dates to be in NZ timezone taking daylight savings into account.
- Remove milliseconds from example datetimes.
- Remove reference to 'vender' in Camera extension Readme (probably 'veder' or 'veeder' but not meaningful to other organisations.
- Added tests for Scanning extension.
- Specify date-time format in Scanning extension to match STAC schema.

### CODE REVIEW

File: extensions/aerial-photo/examples/item.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
   "stac_version": "1.0.0",
-  "stac_extensions": ["https://linz.github.io/stac/_STAC_VERSION_/aerial-photo/schema.json"],
+  "stac_extensions": ["https://stac.linz.govt.nz/_STAC_VERSION_/aerial-photo/schema.json"],
   "type": "Feature",
   "id": "72360",
   "geometry": null,
   "properties": {
-    "datetime": "1952-04-23T00:00:00.000Z",
+    "datetime": "1952-04-22T12:00:00Z",
```
Comment (l0b0) [inline extensions/aerial-photo/examples/item.json:8]: Should the datetime format change be a separate PR? Also, why offset the hour? Looks like some unexpected NZ datetime change.

File: extensions/historical-imagery/tests/historical-imagery_item.test.js [hunk 2]
```diff
@@ -61,23 +61,4 @@ o.spec('historical-imagery item', () => {
       ),
     ).equals(true)(JSON.stringify(validate.errors));
   });
-
```
Comment (l0b0) [inline extensions/historical-imagery/tests/historical-imagery_item.test.js:64]: Why delete this? It's not explained in the commit message.

File: extensions/scanning/tests/scanning.test.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
+import o from 'ospec';
```
Comment (l0b0) [inline extensions/scanning/tests/scanning.test.js:1]: Do these tests belong in a separate PR?

**Review by l0b0 — CHANGES_REQUESTED** [review]

**Review by paulfouquet — APPROVED** [review]

File: extensions/historical-imagery/tests/historical-imagery_item.test.js (hunk 2 above)
Comment (amfage) [reply extensions/historical-imagery/tests/historical-imagery_item.test.js:64]: This test is deleted because the eo:bands field is no longer required by the historical imagery extension.

File: extensions/aerial-photo/examples/item.json (hunk 1 above)
Comment (amfage) [reply extensions/aerial-photo/examples/item.json:8]: The offset is because all dates should be in UTC but the source is in NZ time.

File: extensions/historical-imagery/tests/historical-imagery_item.test.js (hunk 2 above)
Comment (l0b0) [reply extensions/historical-imagery/tests/historical-imagery_item.test.js:64]: I see, the "Changes" stuff in the PR description should ideally be in the commit message for future reference.

File: extensions/aerial-photo/examples/item.json (hunk 1 above)
Comment (l0b0) [reply extensions/aerial-photo/examples/item.json:8]: Ah right. These changes all seem unrelated enough that they should ideally be a bunch of PRs. What do you think, @blacha?

Comment (amfage) [reply extensions/aerial-photo/examples/item.json:8]: Sure, let me know how you would like me to split them up.

Comment (l0b0) [reply extensions/aerial-photo/examples/item.json:8]: I'd normally just create a new branch from origin/master for each independent change and commit/push them as a PR each. Basically, you might end up with one PR to change the URLs to stac.linz.govt.nz, another to not require eo:bands asset field in the Historical Imagery extension, etc. That way, even years from now someone could understand each change in isolation. It also helps when we sometimes have to revert a commit. If the commits are small and self-contained this is trivial, if they include a bunch of different changes it's really difficult to know which have to be reverted.

Comment (amfage) [reply extensions/aerial-photo/examples/item.json:8]: Will do, thanks! Ignore this PR for now.

**Comment by amfage** [comment]
Split into separate PRs:
#145, #146, #147, #148
